docs/section9/Rule6-la.md

```diff
@@ -0,0 +1,33 @@
+
+# Lighting - Rule 6-la
+
+**Rule ID:** 6-la  
+**Rule Description:** Interior lighting level for spaces with plug in lighting not shown in design drawings, LPD shall be greater of as-designed or prescriptive values in Table 9.6.1. For the dwelling units, lighting power used in the simulation shall be equal to 0.60 W/ft2 or as designed, whichever is greater. (Note XC, as per the TCD this rule about just checking the dwellling unit?)  
+**Appendix G Section:** Lighting  
```

Comment:
@xingcx is the rule ID incorrect? should this be 6jl ?

Reply:
> @xingcx is the rule ID incorrect? should this be 6jl ?

Yes you are right. This was originally to check dwelling unit only. Now dwelling unit is covered under Rule 6-jl. This can be deleted. Let me know if we want to separate Rule 6j (for hotel/model and dorm) and Rule 6l (for dwelling unit).

Reply:
@xingcx when you add the new pull request, please link to this discussion.